web/src/js/ducks/flows.js

```diff
@@ -144,15 +144,21 @@ export function upload(file) {
     return dispatch => fetchApi('/flows/dump', { method: 'post', body })
 }
 
-
-export function select(id) {
-    return {
-        type: SELECT,
-        flowIds: id ? [id] : []
+/**
+ * @public
+ */
+export function select(id, selectedFlowIds, ctrl) {
```

Comment:
Can we split this into `select` and `ctrlSelect`/ `shiftSelect` so that other parts can still call `select` without suppling all the other arguments?
(There's also a merge conflict from @cle1000's work I think)